projects/components/src/common/interfaces/action-item.interface.ts

```diff
@@ -180,9 +180,9 @@ export interface ContextualActionInlineDisplayConfig {
     styling: ActionStyling.INLINE;
     /**
      * If the title should be the button label, icon, or both
-     * Defaults to ICON if unset.
+     * Defaults to {@link TextIcon.TEXT} when unset
      */
-    buttonContents: TextIcon;
+    buttonContents?: TextIcon;
```

Comment:
Can we add a test for these defaults? Just a test where you set `buttonContents: undefined` and then assert that it's displaying as TEXT?